docs/content/latest/architecture/transactions/read-committed.md

```diff
@@ -0,0 +1,785 @@
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td><code> commit;</code>
+   </td>
+  </tr>
+  <tr>
+   <td>INSERT 0 1
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td><code> select * from test;</code>
+    <p>
+     k |  v
+    <br />
+    ---+-----
+    <br />
+     2 | 100
+    <br />
+    (1 row)
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td><code>commit;</code>
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+**iii) INSERT old key that is removed by other transaction**
+
+```sql
+truncate table test;
+insert into test values (1, 1);
+```
+
+<table>
+  <tr>
+   <td><code>begin transaction isolation level read committed;</code>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td><code>begin transaction isolation level read committed;</code>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td><code>update test set k=2 where k=1;</code>
+    <p> UPDATE 1
+   </td>
+  </tr>
+  <tr>
+   <td><code>insert into test values (1, 1);</code>
+<p>
+<strong>(waits)</strong>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td><code> commit;</code>
+   </td>
+  </tr>
+  <tr>
+   <td>INSERT 0 1
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td><code> select * from test;</code>
+    <p>
+     k | v
+    <br />
+    ---+---
+    <br />
+     1 | 1
+    <br />
+     2 | 1
+    <br />
+    (2 rows)
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td><code> commit;</code>
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+**iv) same with iii) but with ON CONFLICT**
+
+```sql
+truncate table test;
+insert into test values (1, 1);
+```
+
+<table>
+  <tr>
+   <td><code>begin transaction isolation level read committed;</code>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td><code>begin transaction isolation level read committed;</code>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td><code> update test set k=2 where k=1;</code>
+    <p> UPDATE 1
+   </td>
+  </tr>
+  <tr>
+   <td><code> insert into test values (1, 1) on conflict (k) do update set v=100; </code>
+  <p> <strong>(waits)</strong>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+   <td><code> commit;</code>
+   </td>
+  </tr>
+  <tr>
+   <td>INSERT 0 1
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td><code> select * from test;</code>
+    <p>
+     k |  v
+    <br />
+    ---+-----
+    <br />
+     1 |  1
+    <br />
+     2 |  1
+    <br />
+    (2 rows)
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td><code> commit;</code>
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+### 5 Cross feature interaction
+
+This feature has interaction with these features:
+
+1. **Follower reads (integration in progress):** When follower reads is turned on, the read point for each statement in a READ COMMITTED transaction will be picked as _Now() - yb_follower_read_staleness_ms _ (if the transaction/statement is known to be explicitly/ implicitly read only).
```

Comment:
_Now() - yb_follower_read_staleness_ms _ - this doesn't format correctly.

Reply:
done